src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruits = new HashMap<>();
+    private static final int INITIAL_QUANTITY = 0;
+
+    public static void addFruit(String fruit, int quantity) {
+        fruits.put(fruit, fruits.getOrDefault(fruit, INITIAL_QUANTITY) + quantity);
+    }
+
+    public static void removeFruit(String fruit, int quantity) {

```

Comment:
The method `removeFruit` should check if the fruit exists in the storage before attempting to remove it. If the fruit does not exist, it should throw an exception to indicate that the operation cannot be performed.